### PR TITLE
added doc for schedulers.py

### PR DESCRIPTION
**Pull request by bashirk**
## Description
This PR adds inline documentation to the schedulers.py file.

## Relevant Issue
Issue #269 

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/NimbleEdge/EnvisEdge/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/NimbleEdge/EnvisEdge/blob/main/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [NimbleEdge Styleguide](https://numpydoc.readthedocs.io/en/latest/format.html)

### CODE REVIEW

File: fedrec/optimization/schedulers.py [hunk 1]
```diff
@@ -6,6 +6,34 @@
...
 @registry.load('lr_scheduler', 'dlrm')
 class LRPolicyScheduler(_LRScheduler):
+    """
+    This class is used to adjust the learning rate of the optimizer during
+    training. It implements the LRPolicy, which is a learning rate decay
+    policy described in the paper "Lazy Layer Scheduling for Training of
+    Deep Neural Networks."
+    
+    The LRPolicy is designed to decrease the learning rate after a fixed
+    number of epochs, and increase it after a fixed number of epochs.
```
Comment (divyanshugit) [inline fedrec/optimization/schedulers.py:16]: It's okayish. Whenever you write the name of the paper please write the name of the paper as a hyperlink. You can read more about LR Schedulers from [here](https://d2l.ai/chapter_optimization/lr-scheduler.html) and It's a wrapper around the LRScheduler in Pytorch

**Review by divyanshugit — CHANGES_REQUESTED** [review]

File: fedrec/optimization/schedulers.py (hunk 1 above)
Comment (bashirk) [reply fedrec/optimization/schedulers.py]: Thanks. I've updated the doc.

File: fedrec/optimization/schedulers.py [hunk 2]
```diff
@@ -6,6 +6,37 @@
 
 @registry.load('lr_scheduler', 'dlrm')
 class LRPolicyScheduler(_LRScheduler):
+    """
+    This class is used to adjust the learning rate of the optimizer during
```
Comment (divyanshugit) [inline fedrec/optimization/schedulers.py:10]:
```suggestion
    LRPolicyScheduler is used to adjust the learning rate of the optimizer during
```

**Review by divyanshugit — CHANGES_REQUESTED** [review]

**Comment by divyanshugit** [comment]
LGTM!! @ramesht007
